packages/webview_flutter/webview_flutter/example/ios/RunnerTests/FLTWebViewTests.m

```diff
@@ -11,19 +11,215 @@
 
 static bool feq(CGFloat a, CGFloat b) { return fabs(b - a) < FLT_EPSILON; }
 
-@interface FLTWebViewTests : XCTestCase
+@interface FLTWebViewController (Test)
+- (NSURLRequest *)buildNSURLRequest:(NSDictionary<NSString *, id> *)arguments;
+- (void)onLoadRequest:(FlutterMethodCall *)call result:(FlutterResult)result;
+- (instancetype)initWithFrame:(CGRect)frame
+               viewIdentifier:(int64_t)viewId
+                    arguments:(id _Nullable)args
+              binaryMessenger:(NSObject<FlutterBinaryMessenger> *)messenger;
+@end
 
-@property(strong, nonatomic) NSObject<FlutterBinaryMessenger> *mockBinaryMessenger;
+@interface MockFLTWKWebView : FLTWKWebView
+@property(nonatomic, nullable) NSMutableURLRequest *receivedResult;
+@end
+
+@implementation MockFLTWKWebView
+
+- (WKNavigation *)loadRequest:(NSMutableURLRequest *)request {
+  _receivedResult = request;
+  return nil;
+}
+
+@end
+
+@interface MockFLTWebViewController : FLTWebViewController
+@end
+
+@implementation MockFLTWebViewController {
+  MockFLTWKWebView *mockFLTWKWebView;
+}
+
+- (FLTWKWebView *)createFLTWKWebViewWithFrame:(CGRect)frame
+                                configuration:(WKWebViewConfiguration *)configuration
+                           navigationDelegate:(FLTWKNavigationDelegate *)navigationDelegate {
+  mockFLTWKWebView = [MockFLTWKWebView new];
+  return mockFLTWKWebView;
+}
+
+- (MockFLTWKWebView *)getResultObject {
```

Comment:
ObjC methods should not start with `get`.

Reply:
It's also not clear to me why this is a "result object" when it's created by the mock itself. Why not just call this `mockWebView` (and make it a property)?